### DISCUSSION TITLE
非别名路径如何配置nginx

### DISCUSSION THREAD

**Discussion by zFitness** [umijs/dumi#1993] [General]
现在我通过修改 .dumirc.ts 中的 base 和 publicPath, dev 环境已经可以正常访问了。
```
import { defineConfig } from "dumi";

export default defineConfig({
  outputPath: "docs-dist",
  mfsu: false,
  base: "/da",
  publicPath: '/da/',
  themeConfig: {
    name: "Cats"
  },
});
```
但是配置 nginx 不对：
```
server {
  listen 80 default_server;
  server_name _;
  
  # html不缓存
  location ^~/dependency-analysis {
    index index.html;
    alias /data/sites/cats;
  }
}
```

点击菜单跳转的链接没有带上/dependency-analysis前缀

**Comment by PeachScript** [comment]
菜单跳转链接是指什么，是自己配的还是约定式生成的

另外 base 应当以 `/` 结尾，参考：https://d.umijs.org/guide/faq#%E9%9D%9E%E6%A0%B9%E7%9B%AE%E5%BD%95%E9%83%A8%E7%BD%B2

**Reply by zFitness** [reply]
你好，是这样的，
<img width="374" alt="image" src="https://github.com/umijs/dumi/assets/41500499/de32c748-e9e5-4fcb-8668-178af7edf9ef">
我点击这个菜单没有带上/dependency-analysis前缀

**Reply by PeachScript** [reply]
我在 dumi 仓库设置 base + publicPath，然后本地 serve docs-dist 产物，导航链接是有 base 的；问题可能出在部署环节，检查你部署的产物是否和本地一致
